app/react/components/BoxSelector/BoxSelectorItem.module.css

```
@@ -38,10 +38,11 @@
   padding-left: 20px;
 }
 
+/* used for BE teaser. Dark theme specs defined by EE-5621 */
 .box-selector-item.limited.business label,
 .box-selector-item.limited.business input:checked + label {
   @apply border-warning-7 bg-warning-1 text-black;
-  @apply th-dark:bg-warning-8 th-dark:bg-opacity-10 th-dark:text-white;
+  @apply th-dark:border-blue-8 th-dark:bg-[color:var(--bg-BE-only)] th-dark:bg-opacity-10 th-dark:text-white;
```

```suggestion
  @apply th-dark:border-blue-8 th-dark:bg-[--bg-BE-only] th-dark:bg-opacity-10 th-dark:text-white;
```
https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values:~:text=When%20using%20a%20CSS%20variable%20as%20an%20arbitrary%20value%2C%20wrapping%20your%20variable%20in%20var(...)%20isn%E2%80%99t%20needed%20%E2%80%94%20just%20providing%20the%20actual%20variable%20name%20is%20enough%3A

---

since --bg-BE-only is basically white with some opacity `th-dark:bg-white th-dark-bg-opacity-10` should have similar effect. or `th-dark:bg-white/[0.08]` if you want to specific

---

I missed this line in the tailwind documentation, thanks!
Regarding the color, I applied the one given by design in Jira (#E1DFDF 8%)

---

Well, seems our build process doesn't like this syntax
Reverting these changes.

```
SyntaxError

(48:3) /home/vsts/work/1/s/app/react/portainer/feature-flags/feature-flags.css
The `th-dark:bg-[--bg-BE-only]` class does not exist.
If `th-dark:bg-[--bg-BE-only]` is a custom class, make sure it is defined within a `@layer` directive.

  46 | 
  47 |   @apply bg-warning-5 text-warning-9;
> 48 |   @apply th-dark:bg-[--bg-BE-only] th-dark:text-blue-8;
     |   ^
  49 | 
  50 |   padding: 5px 10px;

```